### PR TITLE
TD Balance Changes

### PR DESCRIPTION
**Pull request by AoAGeneral**
http://www.sleipnirstuff.com/forum/viewtopic.php?f=82&t=17430

List of changes on forums.
#8244 is now out of date due to technical issues. Please close when possible/delete.

Ltnk: Changes
HP 350 from 360
damage vs heavy 90 from 100

Bike: Changes
Price 500 from 600

Artillery: Changes
Chance to explode 0 from 75

Chem Warrior: Changes
reload delay 65 from 70

Genadier: Changes
Grenade projectile speed 140 from 119

Hummer/Buggy: Changes
Damage vs heavy 15 from 20

### CODE REVIEW

File: mods/cnc/rules/vehicles.yaml [hunk 1]
```diff
@@ -154,7 +154,7 @@ ARTY:
 	Explodes:
 		Weapon: UnitExplode
```
Comment (Unit158) [inline mods/cnc/rules/vehicles.yaml]: If you're changing the chance to 0, you should remove Explodes entirely.

Comment (AoAGeneral) [inline mods/cnc/rules/vehicles.yaml]: Aye this is true. I will need to do that. I will copy paste the command line down incase a future issue arises.

**Comment by Unit158** [comment]
After you remove the Explodes tag, :+1:

**Comment by AoAGeneral** [comment]
I think I did that right. let me know =D

**Comment by Mailaender** [comment]
Looks like the crowd in the forums agree with these balancing proposals. :+1:

**Comment by Mailaender** [comment]
Thanks. Your first merged pull request if I am not mistaken. :tada:

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/3eaa02ccddfc6db0d27f49aba32e2e3c7abfa2c5...179baf84d9472ca598f156161e5b986e87701bbc)

**Comment by AoAGeneral** [comment]
Indeed it is and thank you! =D

**Comment by chrisforbes** [comment]
Best add yourself to AUTHORS then.

**Pull request merged**